packages/pwa/app/components/_app/index.jsx

```diff
@@ -300,9 +327,9 @@ App.propTypes = {
     children: PropTypes.node,
     targetLocale: PropTypes.string,
     defaultLocale: PropTypes.string,
-    location: PropTypes.object,
     messages: PropTypes.object,
-    categories: PropTypes.object
+    categories: PropTypes.object,
+    appOrigin: PropTypes.string
```

Comment:
Why is `appOrigin` a prop? It doesn't look like it's ever being used as a prop, e.g. <App appOrigin={'blah'} >

Reply:
@bendvc good catch. Yeah, it's no longer needed as a prop.